Source/Urho3D/IO/MemoryBuffer.cpp

```diff
@@ -54,7 +54,7 @@ MemoryBuffer::MemoryBuffer(PODVector<unsigned char>& data) :
 
 MemoryBuffer::MemoryBuffer(const PODVector<unsigned char>& data) :
     AbstractFile(data.Size()),
-    buffer_(data.Begin().ptr_),
+    buffer_(const_cast< unsigned char * >(data.Begin().ptr_)),

```

Comment:
Remove the extra spaces in the `const_cast` conversion and this PR can be merged.